app/boards/arm/s40nc/s40nc.zmk.yml

```diff
@@ -0,0 +1,11 @@
+file_format: "1"
+id: s40nc
+name: S40NC
+type: board
+arch: arm
+features:
+  - keys
+outputs:
+  - usb
+  - ble
+url: https://mechwild.com
```

Comment:
Do you have a more direct link to this board specifically?

Reply:
Not currently. I will be restructuring the projects section for the site soon and most of the information is hosted in the discord.